Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotModule.java

```diff
@@ -35,8 +35,8 @@ protected void refreshItems() throws AzureCmdException {
     }
 
     @Override
-    public void renderDeploymentSlots(@NotNull final List<DeploymentSlot> slots) {
-        slots.forEach(slot -> addChildNode(new DeploymentSlotNode(
-            this, slot.name(), slot.state(), slot.operatingSystem().toString(), this.subscriptionId)));
+    public void renderDeploymentSlots(final String webAppId, @NotNull final List<DeploymentSlot> slots) {
```

Comment:
no need to pass webappid

Reply:
yes, will get it from `slot.parent().id()`